crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -322,6 +430,7 @@ impl SyncTaskSupervisor {
 
 struct SyncServiceInner {
     encryption_sync_service: Arc<EncryptionSyncService>,
+    with_offline_mode: bool,
```

Comment:
Can you add a tiny doc comment explaining what it is? (notably that it's static for the sync service's lifetime, it won't change value over time)

Reply:
Sure: 0d99447daa2653cdcea16f9ef3fb548ecd5905c7.